src/OrchardCore/OrchardCore.Abstractions/Shell/Extensions/ShellSettingsExtensions.cs

```diff
@@ -0,0 +1,43 @@
+using OrchardCore.Environment.Shell.Models;
+
+namespace OrchardCore.Environment.Shell
+{
+    public static class ShellSettingsExtensions
+    {
+        /// <summary>
+        /// Wether it is the 'Default' tenant or not.
+        /// </summary>
+        public static bool IsDefaultShell(this ShellSettings settings) =>
+            settings is not null && settings.Name == ShellHelper.DefaultShellName;
```

Comment:
You need moar pattern matching love in your life 😃 

```suggestion
            settings is { Name: ShellHelper.DefaultShellName };
```

Reply:
Thanks for the suggestion, beautiful ;)